packages/experimental/Shadow/src/__tests__/Shadow.test.tsx

```diff
@@ -1,172 +1,89 @@
 import * as React from 'react';
 import { Text } from 'react-native';
 import { Shadow } from '../Shadow';
+import { useFluentTheme } from '@fluentui-react-native/framework';
 import * as renderer from 'react-test-renderer';
 import { checkRenderConsistency, checkReRender } from '@fluentui-react-native/test-tools';
 
+interface ShadowTestProps {
+  displayText: string;
+  depth: string;
+}
+
+const TestShadow: React.FunctionComponent<ShadowTestProps> = (props: ShadowTestProps) => {
```

Comment:
Making a separate function component was necessary in order to be get the shadow from the theme. If there's another way let me know.

Reply:
If you wanted, I guess it'd be OK to have a hardcoded test ShadowToken here. I like using the theme better though.